client/state/happychat/actions.js

```diff
@@ -1,82 +1,18 @@
-/**
- *  IMPORTANT NOTE BEFORE EDITING THIS FILE **
- *
- * We're in the process of moving the side-effecting logic (anything to do with connection)
- * into Redux middleware. If you're implementing something new or changing something,
- * please consider moving any related side-effects into middleware.js.
- *
- * @format
- */
+/** @format **/
```

Comment:
Just curious, what does this do?

Reply:
Drive-By answer:

it means the file is being formatted with [calypso-prettier](https://github.com/Automattic/calypso-prettier).

Theres some more background in this issue: https://github.com/Automattic/wp-calypso/issues/12260#issuecomment-332293613